cdc/log.cc

```diff
@@ -1458,8 +1444,53 @@ class transformer final : public change_processor {
       }
     }
 
-    void set_preimage_select_result(lw_shared_ptr<cql3::untyped_result_set> rs) {
-        _preimage_select_result = std::move(rs);
+    // Note: this assumes that the results are from one partition only
+    void load_preimage_results_into_state(lw_shared_ptr<cql3::untyped_result_set> preimage_set, bool static_only) {
```

Comment:
In the future, I'd like to change the way load preimage results to the state, see [my other comment](https://github.com/scylladb/scylla/pull/6741#discussion_r449011530). It will have to be a method of the `transformer` in order to be able to access `_clustering_row_states` and `_static_row_state` fields.